pyscripts/mask.py

```diff
@@ -219,6 +232,127 @@ def _process_data(self, data_map: Any) -> Any:
         return d
 
 
+class PlaintextMask():
```

Comment:
Thank you @stuggi for this, it is very useful, especially if we're going to see more often human operators that use to store sensitive data in `ConfigMaps` (which is not recommended but not forbidden).
I see that tests pass and I'm ok to land this patch as it is and resolve the immediate problem described in the associated Jira.
As a follow up I'd like to deduplicate some code that is in common between the two classes, (e.g. create a `BaseMask` class with the basic methods and inherit from it so we can customize how we process `Secrets` and `ConfigMaps`/`CRs`).
I assume you do not want to go through this kind of changes in this PR, so let me know if we want to land it as it is.